packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -327,7 +363,7 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
     }
 
     if (_state == _ScaleState.started && onUpdate != null)
-      invokeCallback<void>('onUpdate', () => onUpdate(ScaleUpdateDetails(scale: _scaleFactor, focalPoint: _currentFocalPoint, rotation: _computeRotationFactor())));
+      invokeCallback<void>('onUpdate', () => onUpdate(ScaleUpdateDetails(scale: _scaleFactor, horizontalScale: _horizontalScaleFactor, verticalScale: _verticalScaleFactor, focalPoint: _currentFocalPoint, rotation: _computeRotationFactor())));
```

Comment:
this line is very long. Maybe add some line breaks to make it more readable?

Reply:
Is this a good compromise between brevity and dartfmt?
```
invokeCallback<void>(
    'onUpdate',
    () => onUpdate(
        ScaleUpdateDetails(scale: _scaleFactor, horizontalScale: _horizontalScaleFactor,
            verticalScale: _verticalScaleFactor, focalPoint: _currentFocalPoint,
            rotation: _computeRotationFactor(),
        ),
    ),
);
```

Reply:
It would probably be better to have each property on a separate line (and since this is multiline now you'll have to replace the `() =>` with `() { }` as per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods.

I would format it as follows:

```dart
invokeCallback<void>('onUpdate', () { 
  onUpdate(ScaleUpdateDetails(
    scale: _scaleFactor, 
    horizontalScale: _horizontalScaleFactor,
    verticalScale: _verticalScaleFactor,
    focalPoint: _currentFocalPoint,
    rotation: _computeRotationFactor(),
  )),
});
```